assets/stylesheets/shipping-label.scss

```diff
@@ -66,4 +65,96 @@
 		margin-bottom: 4px;
 		text-transform: uppercase;
 	}
+
+	.gridicon.is-success {
+		color: $alert-green;
+	}
+
+	.is-warning {
+		color: $alert-yellow;
+	}
+
+	.is-error {
+		color: $alert-red;
+	}
+
+	.suggestion-container {
+		display: flex;
+		align-items: flex-start;
+
+		.suggestion-title {
+			font-weight: bold;
+		}
+
+		.suggestion {
+			border: 1px solid transparent;
+			border-radius: 4px;
+			padding: 16px;
+			flex-grow: 1;
+
+			&:first-child {
+				margin-right: 16px;
+			}
+
+			&.is-selected {
+				border-color: $blue-medium;
+			}
+		}
+
+		.summary {
+			margin: 8px 0 8px 24px;
+			font-weight: normal;
+
+			p {
+				margin-bottom: 2px;
+			}
+		}
+
+		.suggestion-edit-button {
+			margin-left: 24px;
+			text-transform: uppercase;
+		}
+	}
+}
+
+// Address
+
+.address__city-state-postal-code {
+	display: flex;
+	flex-direction: column;
+
+	@include breakpoint( ">960px" ) {
+		flex-direction: row;
+	}
+}
+
+.address__address-1.form-fieldset {
+	margin-bottom: 8px;
+}
+
+.address__city,
+.address__state,
+.address__postal-code {
+	width: 100%;
+
+	@include breakpoint( ">960px" ) {
+		width: 33%;
```

Comment:
A small nitpick. For some countries there isn't a concept of "State", so for those countries the `State` fieldset simply isn't rendered. So instead of hard-coding here a `33%`, could you use something more general? `display: flex` would work better I think.